Add reorder option to entity selector (#40062)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

  type: boolean
  default: false
  required: false
reorder:
  description: >

>     Allows to reorder entities (in case `multiple` is set to `true`).

Reviewer: ```suggestion
    Allows reordering of entities (only applies if `multiple` is set to `true`).
```
